tests/test_attributes.py

```diff
@@ -59,3 +59,12 @@ def test_update_no_changes() -> None:
 
     z.update_attributes({})
     assert dict(z.attrs) == {"a": [], "b": 3}
+
+
+def test_del_works() -> None:
+    store = zarr.storage.MemoryStore()
+    z = zarr.create(10, store=store, overwrite=True)
```

Comment:
Please change this to `create_array` or `create_group`.

Reply:
used parametrize to test both.